infrastructure/kubernetes/app-backend.yml

```diff
@@ -36,6 +36,19 @@ spec:
         env:
         - name: "NODE_ENV"
           value: "production"
+        - name: "API_URL"
+          value: "https://agid-apim-test.azure-api.net/api/v1"
+        - name: "REDIS_URL"
+          value: "redis://redis"
+        - name: "SAML_CALLBACK_URL"
+          value: "http://app-backend.k8s.test.cd.teamdigitale.it/assertionConsumerService"
+        - name: "SAML_ISSUER"
+          value: "http://app-backend.cd.teamdigitale.it"
```

Comment:
this URL doesn't exist

Reply:
this has not to be an url but a unique identifier (so usually it is the url of the Service Provider). I suppose this will be the final (production) url of the proxy